parm/archive/gefs_ice.yaml.j2

```
@@ -0,0 +1,36 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".ice.t" + cycle_HH + "z." %}
+
+gefs_ice:
+    name: "GEFS_ICE"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_ice.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% for mem in membr %}
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${MEMDIR}':mem }) %}
+
+    {% set COMIN_ICE_HISTORY = COM_ICE_HISTORY_TMPL | replace_tmpl(tmpl_dict) %}
+
+    # Select netcdf files to copy to the atardir
+    {% if RUN == "gefs" %}
+        {% if path_exists(COMIN_ICE_HISTORY) %}
+            {% for fhr in range(FHMIN_GFS + FHOUT_ICE_GFS, FHMAX_GFS + FHOUT_ICE_GFS, FHOUT_ICE_GFS) %}
+                {% set file_name = head ~ "6hr_avg" ~ ".f" ~ '%03d'|format(fhr) ~ ".nc" %}
+                {% set file_set = COMIN_ICE_HISTORY ~ "/" ~ file_name %}
+        - "{{ file_set | relpath(ROTDIR)}}"
```

Since this is a single file, suggest changing the name to something that identifies it as such:

```suggestion
                {% set file_path = COMIN_ICE_HISTORY ~ "/" ~ file_name %}
        - "{{ file_path | relpath(ROTDIR)}}"
```